crates/rooch/src/commands/account/commands/list.rs

```diff
@@ -50,15 +51,18 @@ impl LocalAccountView {
     }
 }
 
-#[derive(Serialize, Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct AccountView {
+    #[serde(flatten)]
```

Comment:
If we `flatten` the `local_account`, the json path in `cmd.feature` must also be updated.